Controller/ConnectController.php

```diff
@@ -34,6 +34,13 @@
  */
 class ConnectController extends ContainerAware
 {
+
+    protected function hasUser()
```

Comment:
Protected function `hasUser` should be on end of class with `getTokenStorage` to keep coherence.

Could also be useful to add a BC notice (with which SF version is concerned) on comment to not forget to remove it later.